### PR TITLE
Expose select2 l10n

### PR DESCRIPTION
**Pull request by unho**

### CODE REVIEW

File: pootle/apps/pootle_app/assets.py [hunk 1]
```diff
@@ -50,6 +54,18 @@
 register('js_editor', js_editor)
 
 
+rel_path = os.path.join('js', 'node_modules', 'select2', 'dist', 'js', 'i18n')
+select2_l10n_dir = os.path.join(settings.WORKING_DIR, 'static', rel_path)
+l10n_files = [os.path.join(rel_path, f)
+              for f in os.listdir(select2_l10n_dir)
+              if (os.path.isfile(os.path.join(select2_l10n_dir, f)) and
```
Comment (phlax) [inline pootle/apps/pootle_app/assets.py]: can we put and/or operators at beginning of line please

**Comment by dwaynebailey** [comment]
@unho the first commit IIUC is completely uneeded.  This will get overrtten/created when you run `make assets` and there is no need to keep this in our repo.

File: Makefile [hunk 2]
```diff
@@ -44,6 +45,7 @@ travis-assets:
 		cd ${JS_DIR} && \
 		npm install && \
 		cd ${CWD}; \
+	    cp ${JS_DIR}/node_modules/select2/dist/js/i18n/*.js ${JS_DIR}/select2_l10n/; \
```
Comment (dwaynebailey) [inline Makefile]: Please use whatever whitespace the file is using

File: pootle/apps/pootle_app/assets.py [hunk 3]
```diff
@@ -50,6 +54,19 @@
 register('js_editor', js_editor)
 
 
+rel_path = os.path.join('js', 'select2_l10n')
+select2_l10n_dir = os.path.join(settings.WORKING_DIR, 'static', rel_path)
```
Comment (dwaynebailey) [inline pootle/apps/pootle_app/assets.py]: Just curious, if we are already using this in static/ dir.  Why can't we just pick these out of node_modules dir in static dir?

Comment (unho) [inline pootle/apps/pootle_app/assets.py]: When we build the packages (for PyPI), the `node_modules` is not present. That is the reason why we rely on this new `select2_l10n` subdirectory within `static/js/` directory.

**Pull request merged**